### ISSUE TITLE
Add support for .ixx files (C++ modules)

### ISSUE THREAD

**Issue by ruifig** [slynch8/10x#1284]
Just downloaded 10x to give it a go for the first time  (I'm impressed).
Opened a project I've been working on to test it out, and noticed that it has no support for C++ modules. 
Things I *think* are missing

* .ixx files don't get syntax highlighted. 
* .ixx files probably not being parsed at all, since code completion doesn't discover anything coming from modules?

Only spent a few minutes with 10x so far, so maybe I missed some settings.

**Comment by slynch8** [comment]
You're right, I haven't added support for modules yet. However, you can add the .ixx extension to the `ParserExtensions` setting to get them to parse.

**Comment by ruifig** [comment]
> You're right, I haven't added support for modules yet. However, you can add the .ixx extension to the `ParserExtensions` setting to get them to parse.

I see. A quick test, and I can confirm it works. Also, seems like code completion also works ?
What's missing then, since you said "no support for modules yet" ?

**Comment by slynch8** [comment]
That's good. I'm not sure to be honest, I've never used modules. If it all works that's great.
